### PR TITLE
[IMP] events: update track and manage talks

### PR DESCRIPTION
**Pull request by ksc-odoo**
PROJECT TASK: https://www.odoo.com/web#id=3915122&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

This PR discuss what the Talks, Talk Proposals, and Agenda event website pages entail -- and how to utilize and navigate them.
This is meant as a supplemental PR to the 'Create event tracks' doc...which is focused more on the backend elements and how to create tracks for an event.

**This PR should be published/merged after the 'event_tracks' PR
That's why there is likely going to be continued Build Errors -- because that 'event_tracks' doc is referenced in the seealso at the end.**

I worked off the original RST file name so that I didn't ruin any potential reference links that could be lingering elsewhere...

### CODE REVIEW

**Comment by ksc-odoo** [comment]
Hey @brse-odoo -- this one is ready for Peer Review whenever you get a chance...Thanks! 👍

File: content/applications/marketing/events/track_manage_talks.rst [hunk 1]
```diff
@@ -1,161 +1,147 @@
...
 
-The :guilabel:`Cohort` view lays out attendee data to better analyze the number of registration
-dates.
+Talk Proposals page
+-------------------
 
-.. note::
-   Tickets sold through sales orders validate attendees as soon as the quotation is confirmed.
+The :guilabel:`Talk Proposals` link takes attendees to a page on the event website, wherein they can
+formerly submit a proposal for a talk (track) for the event, via a custom online form.
```
Comment (brse-odoo) [inline content/applications/marketing/events/track_manage_talks.rst:95]:
```suggestion
formerly submit a proposal for a talk (:dfn:`track`) for the event, via a custom online form.
```
Optional, I just think dfn tags are cool for stuff like this

File: content/applications/marketing/events/track_manage_talks.rst [hunk 2]
```diff
@@ -1,161 +1,147 @@
...
-can be found, along with pertinent data related to those rules.
+If accepted, the internal user can then move the track to the next appropriate stage in the Kanban
+pipeline on the :guilabel:`Event Tracks` page for the event. Then, they will open that track form,
+and click the :guilabel:`Go to Website` smart button to reveal that track's page on the event
+website.
 
-.. image:: track_manage_talks/events-lead-generation-rule-page.png
-   :align: center
-   :alt: How the Lead Generation Rule page looks in Odoo Events.
+From there, they will toggle the :guilabel:`Unpublished` switch in the header to
```
Comment (brse-odoo) [inline content/applications/marketing/events/track_manage_talks.rst:125]:
```suggestion
From there, they can toggle the :guilabel:`Unpublished` switch in the header to
```

File: content/applications/marketing/events/track_manage_talks.rst [hunk 3]
```diff
@@ -1,161 +1,147 @@
...
+company, click the :guilabel:`Events` header option, and select the desired event to view that
+event's website.
 
-When that feature is enabled, two more options become available: *Live Broadcast* and *Event
-Gamification*.
+On the event website, there is an event-specific subheader menu with different options to choose
+from.
 
-:guilabel:`Live Broadcast` allows for the airing of tracks online through a YouTube integration.
+With the :guilabel:`Schedule & Tracks` setting enabled in the Odoo *Events* app, the following links
```
[the remark this answers is not in the document]
Comment (ksc-odoo) [reply content/applications/marketing/events/track_manage_talks.rst:20]: Great catch. You are 100% correct. Changing that now.

File: content/applications/marketing/events/track_manage_talks.rst [hunk 4]
```diff
@@ -1,161 +1,147 @@
...
+are automatically added to the sub-header menu, located on the event website: :guilabel:`Talks`,
```
[the remark this answers is not in the document]
Comment (ksc-odoo) [reply content/applications/marketing/events/track_manage_talks.rst:21]: Good call 👍

**Comment by ksc-odoo** [comment]
Thanks for the super-helpful feedback, @brse-odoo -- made all your suggested edits 👍 This one is now ready for Final Review @StraubCreative -- whenever you get a chance. Thanks! 🙏

**Comment by StraubCreative** [comment]
Rebase to latest on [d772c91](https://github.com/odoo/documentation/pull/9223/commits/d772c91ad9feb870acd869307a9ab439bed7950d)

File: content/applications/marketing/events/track_manage_talks.rst [hunk 5]
```diff
@@ -1,161 +1,148 @@
...
 
-Manage registrations
---------------------
+.. image:: track_manage_talks/talk-proposals-page.png
+   :align: center
+   :alt: The Talk Proposals page on the event website built with the Odoo Events application.
 
-Upon selecting an attendee, Odoo reveals that specific attendee's detail form.
+In addition to the form, an introduction to the page, along with any other pertinent information
+related to the types of talks the event will feature can be added, as well, if needed.
```
Comment (StraubCreative) [inline content/applications/marketing/events/track_manage_talks.rst:103]: Redundant (in addition, as well)
```suggestion
In addition to the form, an introduction to the page, along with any other pertinent information
related to the types of talks the event will feature can be added, if needed.
```

File: content/applications/marketing/events/track_manage_talks.rst [hunk 6]
```diff
@@ -1,161 +1,148 @@
...
+If an attendee wishes to favorite a track, they can click the :icon:`fa-bell-o` :guilabel:`(empty
+bell)` icon, located to the right of the track title. Attendees will know a track has been favorited
+when they notice the icon has been changed to :icon:`fa-bell` :guilabel:`(filled bell)` icon.
 
-Lastly, in the :guilabel:`Lead Default Values` section, designate a :guilabel:`Lead Type`, then
-assign it to a specific :guilabel:`Sales Team` (and/or :guilabel:`Salesperson`), and attach tags to
-the rule, if necessary.
+.. seealso::
+   - :doc:`event_tracks`
+   - :doc:`event_essentials`
```
Comment (StraubCreative) [inline content/applications/marketing/events/track_manage_talks.rst:148]: Can remove since they're broken links and are causing CI checks to fail.

**Review by StraubCreative — APPROVED** [review]
Hi @ksc-odoo 

Looks good, I like how you focused the content to be more specific to talks instead of the other topics that were included in the previous iteration (Attendees, LGR, etc.)

I just had a few comments below for you to look at. Once they're addressed to your liking feel free to move on to tech review.

Cheers!

File: content/applications/marketing/events/track_manage_talks.rst [hunk 7]
```diff
@@ -1,161 +1,148 @@
...
-Track and manage talks
-======================
+=================================================
+Talks, talk proposals, and agenda (event website)
+=================================================
 
-With Odoo Events, it's possible to provide attendees with the power to propose presenters to speak
-at events.
+With Odoo *Events*, users can utilize a fully-integrated event website, where attendees can quickly
+access various tracks (talks, presentations, etc.), view entire agendas, and even propose talks for
```
[the remark this answers is not in the document]
Comment (ksc-odoo) [reply content/applications/marketing/events/track_manage_talks.rst:6]: Completely agree. Great call. 👍

File: content/applications/marketing/events/track_manage_talks.rst [hunk 8]
```diff
@@ -1,161 +1,148 @@
...
-
-Talks, talk proposals, and agenda
-=================================
-
-Once those two features are enabled, the following links are automatically added to the sub-header
-menu, located on the event page on the website: :guilabel:`Talks`, :guilabel:`Talk Proposals`, and
-:guilabel:`Agenda`. Any attendee can freely access these menu items and their corresponding content.
+   To enable the :guilabel:`Schedule & Tracks` setting, navigate to :menuselection:`Events app -->
+   Configuration --> Settings`, tick the checkbox beside :guilabel:`Schedule & Tracks`, and click
+   :guilabel:`Save`.
```
[the remark this answers is not in the document]
Comment (ksc-odoo) [reply content/applications/marketing/events/track_manage_talks.rst:31]: You are absolutely right! Changing this right now. Thanks for pointing this out! 🙏

**Comment by ksc-odoo** [comment]
Thanks again for the awesome feedback/suggestions, @StraubCreative -- really appreciate it! I made all those necessary adjustments, and believe this one is now ready for Tech Review, @samueljlieber 🫡

File: content/applications/marketing/events/track_manage_talks.rst [hunk 9]
```diff
@@ -1,161 +1,143 @@
...
 
-Upon selecting an attendee, Odoo reveals that specific attendee's detail form.
+In addition to the form, an introduction to the page, along with any other pertinent information
+related to the types of talks the event will feature can be added, if needed.
 
-From here, event badges can be sent manually, by selecting :guilabel:`Send By Email`. The
-:guilabel:`Attendee` can also be marked as :guilabel:`Attended`, or the registration can be
-canceled altogether via the :guilabel:`Cancel Registration` button.
+The talk propsal form can be modified in a number of different ways, via the web builder tools,
+accesible by clicking :guilabel:`Edit` while on the specific page.
```
Comment (samueljlieber) [inline content/applications/marketing/events/track_manage_talks.rst:105]: Typos :)
```suggestion
The talk proposal form can be modified in a number of different ways, via the web builder tools,
accessible by clicking :guilabel:`Edit` while on the specific page.
```

**Review by samueljlieber — APPROVED** [review]
Hi @ksc-odoo! Nice work on this full rewrite of the events tracks documentation. Your improvements look great, approving with a quick fix! Thanks!
.....
@robodoo delegate=ksc-odoo

**Comment by ksc-odoo** [comment]
Thanks for the feedback, @samueljlieber -- made that quick adjustment, and believe this one is ready to go. Thanks again! 👍 

@robodoo r+

**Pull request closed without merging**